js/src/redux/providers/personalReducer.js

```diff
@@ -25,16 +25,16 @@ export default handleActions({
     const accounts = {};
     const contacts = {};
 
-    Object.keys(accountsInfo).forEach((address) => {
-      const account = accountsInfo[address];
-      const { name, meta, uuid } = account;
-
-      if (uuid) {
-        accounts[address] = { address, name, meta, uuid };
-      } else {
-        contacts[address] = { address, name, meta };
-      }
-    });
+    Object.keys(accountsInfo)
+      .map((address) => Object.assign({}, accountsInfo[address], { address }))
+      .filter((account) => !account.meta.deleted)
+      .forEach((account) => {
+        if (account.uuid) {
```

Comment:
Lemme just check that again, they didn't used to be the same - I think I probably went only half-way with cleaning/simplifying things then.